### DISCUSSION TITLE
Welcome to SailScores Discussions!

### DISCUSSION THREAD

**Discussion by jbfraser1** [SailScores/SailScores#189] [General]
## 👋 Welcome!
  We’re going to see if GitHub Discussions will give us a place to connect. We hope that you:
  * Ask questions you’re wondering about.
  * Let us know of problems you've found with SailScores.
  * Share ideas.
  * Keep things kind and welcoming.

**Comment by DavidGeeraerts** [comment]
[Budd Inlet](https://en.wikipedia.org/wiki/Budd_Inlet) [Star](https://starclass.org/) Fleet Olympia, WA is using SailScores! 
:wink:

**Reply by DavidGeeraerts** [reply]
@jbfraser1 , How does one go about getting write access to:  [Budd Inlet Star Fleet](https://www.sailscores.com/D6BISF)?
Does the person (Patrick) who requested the site have admin ability to setup roles, or something more simple such as "make editor"?
Thank you for your help.
![image](https://github.com/user-attachments/assets/e0f47dbd-843a-4603-8b19-b52b84706144)

**Reply by jbfraser1** [reply]
Any club scorekeeper can add more scorekeepers for that club. The last section of the Club Admin page manages these.
A scorekeeper can be added here by email address, and it will indicate if they have registered on the site. You can add them either before or after they have registered for an account.

![image](https://github.com/user-attachments/assets/06980874-1e74-44eb-a4b0-b3ddf7dc1b7f)

In your case, you're right, Patrick is the only account with permissions for BISF currently. If he's having trouble accessing the site or adding you, let me know. Certainly feel free to email me at jamie@widernets.com
